src/features/distribute/components/Distributions.tsx

```diff
@@ -41,6 +41,8 @@ export function Distributions() {
     );
   }
 
+  console.log("distributionResult", distributionResult);
+
```

Comment:
is this console.log intentional?

Reply:
yes. will cleanup once we make sure entire flow works